### PR TITLE
Rad-222 Make RadiologyOrderValidatorComponentTest unit test

### PR DESCRIPTION
**Pull request by joehiggs**
see https://issues.openmrs.org/browse/RAD-222

### CODE REVIEW

File: api/src/test/java/org/openmrs/module/radiology/order/RadiologyOrderValidatorTest.java [hunk 1]
```diff
@@ -11,70 +11,72 @@
...
-import org.openmrs.test.Verifies;
 import org.springframework.validation.BindException;
 import org.springframework.validation.Errors;
-import org.springframework.validation.ObjectError;
 
 /**
  * Tests methods on the {@link RadiologyOrderValidator} class.
  */
-public class RadiologyOrderValidatorComponentTest extends BaseModuleContextSensitiveTest {
+public class RadiologyOrderValidatorTest extends BaseModuleContextSensitiveTest {
```
Comment (teleivo) [inline api/src/test/java/org/openmrs/module/radiology/order/RadiologyOrderValidatorTest.java]: No need to extend BaseModuleContextSensitiveTest
This makes it a component test, loads in memory db.

File: api/src/test/java/org/openmrs/module/radiology/order/RadiologyOrderValidatorTest.java [hunk 2]
```diff
@@ -86,102 +88,129 @@ public void validate_shouldFailValidationIfConceptIsNull() throws Exception {
...
-	@Verifies(value = "should fail validation if orderer is null", method = "validate(Object, Errors)")
-	public void validate_shouldFailValidationIfOrdererIsNull() throws Exception {
+	public void validate_shouldFailValidationIfDateActivatedAfterDateStopped() throws Exception {
 		
 		RadiologyOrder radiologyOrder = new RadiologyOrder();
 		radiologyOrder.setConcept(Context.getConceptService()
 				.getConcept(88));
 		radiologyOrder.setPatient(Context.getPatientService()
 				.getPatient(2));
+		radiologyOrder.setOrderer(Context.getProviderService()
```
Comment (teleivo) [inline api/src/test/java/org/openmrs/module/radiology/order/RadiologyOrderValidatorTest.java]: You cannot use context, this hits the db. You need to create simple POJOs

**Comment by teleivo** [comment]
@joehiggs I recommend you create a helper method in the test class which creates and returns a RadiologyOrder you can then use in your tests.

**Comment by joehiggs** [comment]
Added the changes--the component test vs unit test distinction makes more sense now. Thanks

**Comment by teleivo** [comment]
@joehiggs can you please squash your 2 commits into one and make sure to run `mvn clean package` before pushing and check if there are any changes you need to add to your commit. because if I run this command I get a diff with formatting changes in the javadocs at RadiologyOrderValidatorTest.java

**Comment by joehiggs** [comment]
Forgot to run mvn clean package--sorry about that. Just ran it, squashed, and committed

**Comment by teleivo** [comment]
thanks @joehiggs, nice work

**Pull request merged**